bcda/models/models.go

```diff
@@ -33,14 +34,43 @@ func InitializeGormModels() *gorm.DB {
 type Job struct {
 	gorm.Model
 	ACO        ACO       `gorm:"foreignkey:ACOID;association_foreignkey:UUID"` // aco
-	ACOID      uuid.UUID `gorm:"primary_key; type:char(36)" json:"aco_id"`
```

Comment:
Was this intentional?

Reply:
It was.  This was never intended to be a primary key for Job.

Reply:
👍